lib/puppet/type/f5_profileserverssl.rb

```diff
@@ -0,0 +1,87 @@
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/parameter/f5_name.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_description.rb'))
+require File.expand_path(File.join(File.dirname(__FILE__),'..','..','puppet/property/f5_truthy.rb'))
+
+
+Puppet::Type.newtype(:f5_profileserverssl) do
+  @doc = 'Manage Server SSL profile objects'
+
+  apply_to_device
+  ensurable
+
+  newparam(:name) do
+    def self.postinit
```

Comment:
I've not come across `postinit` before.  I'd like to understand better what this actually does.  @hunner ?